src/server/api/routers/portal.ts

```
@@ -294,6 +294,13 @@ export const portalRouter = createTRPCRouter({
       where: {
         deletedAt: null,
       },
+      select: {
```

ESLint

[error] Expected object keys to be in ascending order. 'select' should be before 'where'.

(sort-keys)